.github/workflows/trigger-qa-build.yml

```diff
@@ -6,16 +6,76 @@ on:
       workflow:
         description: 'Desired Workflow'
         required: true
-        default: 'LernaPublish'
+        default: 'Lerna Publish'
         type: choice
         options:
-        - LernaPublish
-        - Lerna Publish All Forced
+        - Lerna Publish
+        - Lerna Force Publish Components
+        - Lerna Force Publish Design
+        - Lerna Force Publish Components & Design
+        - Lerna Force Publish All
 
 jobs:
-  trigger-pre-release-build:
+  trigger_pre_release_build:
     name: Trigger Pre-release Build
     runs-on: ubuntu-latest
+    outputs:
+      output1: ${{ steps.summaryFile.outputs.releaseSummary}}
     steps:
-      - name: "Test step"
-        run: echo "test"
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref_name }}
+          # pulls all commits (needed for lerna / semantic release to correctly version)
+          fetch-depth: "0"
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
+          # Defaults to the user or organization that owns the workflow file
+          scope: '@jobber'
+      - name: NPM CI
+        run: npm ci
+      - name: NPM Publish
+        id: npmPublish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.JOBBER_NPM_TOKEN }}
+          ATLANTIS_BRANCH_NAME_RAW: ${{ github.ref_name }}
+          PUBLISH_SETTING: ${{ github.event.inputs.workflow }}
+        run: |
+          ATLANTIS_BRANCH_NAME=$(echo ${ATLANTIS_BRANCH_NAME_RAW:0:10} | sed 's/\///g')
```

Comment:
Generates a preId from the branch name. Removes characters that can't be used in a preId